### PR TITLE
Validate and run prettier in the same npm script

### PR DESCRIPTION
**Pull request by ograu**
So you don't have to worry about it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "test:watch": "jest --watch",
     "start": "grunt serve",
     "prettier": "prettier --write \"src/**/*.js\"",
-    "validate-prettier": "prettier --version; prettier --no-color --check \"src/**/*.js\""
+    "validate-prettier": "npm run prettier && prettier --version; prettier --no-color --check \"src/**/*.js\""
```
Comment (marians) [inline package.json]: Shouldn't that be consistent?

```suggestion
    "validate-prettier": "npm run prettier && npm run prettier --version; npm run prettier --no-color --check \"src/**/*.js\""
```

**Comment by oponder** [comment]
Thx @ograu!  

However there is a `prettier` command already above there. So if you just want to run prettier you can do `npm run prettier`

So this `validate-prettier` is also for the CircleCI step. The idea is that I'd rather not have CircleCI automatically format code, I only want it to fail if the formatting hasn't been done.

So unless I misunderstood, I don't think this change is needed.

**Comment by oponder** [comment]
@ograu I think we can close this one right?

**Comment by ograu** [comment]
Yes, we can close it

**Comment by marians** [comment]
@ograu It's your PR, so you are normally responsible for merging, closing, deleting the branch etc.

**Pull request closed without merging**